Migration script/openIMIS migration v1.4.1 - v1.4.2.sql

```diff
@@ -1514,3 +1514,24 @@ AND LocationType = N'D'
 GO
 
 CREATE UNIQUE CLUSTERED INDEX CI_tblDistricts ON tblDistricts(DistrictId)   
+
+-- OTC-111: Changing the logic of user Roles
+
+IF COL_LENGTH('tblUserRole', 'Assign') IS NULL
+BEGIN
+	ALTER TABLE tblUserRole ADD Assign int NULL
```

Comment:
Please add a default value here to make sure the current users have the role and can assign it.  The same for the demo database.

Reply:
Done. I added a default value in all `sql` files